WordPress/gutenberg
# AligmentMatrixControl: update center cell label to 'Center'

## What?
Updates the label of the center cell of an `AlignmentMatrixControl` from `Center Center` to just `Center`

## Why?
Internally, we refer to the center cell as `center center`. When `center` is passed, it gets converted to `center center`, but that name doesn't make as much sense from an end-user's point of view.

For the label, `Center` is more concise, and better matches how a user would likely describe that cell.

## How?
Updating the strings for both `center` and `center center` alignments in the `AlignmentLabel` record.

## Testing Instructions

- Test the component in Storybook, confirm the center cell is now labelled as `Center`.
- Test the same in the editor on a block like Cover
- Confirm tests still pass

### Testing Instructions for Keyboard
In Storybook, focus on the canvas and press `Tab` once to focus on the control. This should display the current label in a tooltip. Use the arrow keys to change the currently selected cell, navigating away from and then back to `Center`.
